### PR TITLE
[IAMP-700 Update to support Latest Event Log]

### PR DESCRIPTION
**Pull request by egglee**
https://accelbyte.atlassian.net/browse/IAMP-700

Update plugin to support new log standardization

--by Allen.Li

### CODE REVIEW

File: pkg/logger/event/event.go [hunk 1]
```diff
@@ -17,39 +17,46 @@
...
 import (
-	public_source_ip "github.com/AccelByte/public-source-ip"
 	"github.com/emicklei/go-restful"
 	"github.com/fatih/structs"
 	"github.com/sirupsen/logrus"
 )
 
 const (
 	eventLogAttribute     = "EventLog"
-	eventType             = "event"
```
Comment (thoriqsatriya) [inline pkg/logger/event/event.go]: To support the current version of event log, I think we still need this event_type to be "event"

File: pkg/logger/event/event.go [hunk 2]
```diff
@@ -95,23 +100,23 @@ func Log(realm string, fn ExtractAttribute) restful.FilterFunction {
 
 		switch evt.level {
 		case logrus.FatalLevel:
-			log.Fatal(evt.message...)
+			log.Fatal()
```
Comment (thoriqsatriya) [inline pkg/logger/event/event.go]: Why do you remove the message? I think we still need the message, for example for logging error message

File: pkg/logger/event/event.go [hunk 3]
```diff
@@ -17,39 +17,48 @@
...
 	Realm            string                 `structs:"realm"`
-	SourceIP         string                 `structs:"source_ip"`
-	LogType          string                 `structs:"log_type"`
+	topic            string                 `structs:"-"`
+	Action           string                 `structs:"action"`
+	Status           string                 `structs:"status"`
 	level            logrus.Level           `structs:"-"`
-	message          []interface{}          `structs:"-"`
-	additionalFields map[string]interface{} `structs:"-"`
+	Privacy          bool                   `structs:"privacy"`
```
Comment (desnia) [inline pkg/logger/event/event.go]: we don't need this field and to be in the log

File: pkg/logger/event/event.go (hunk 2 above)
Comment (egglee) [reply pkg/logger/event/event.go]: I am sorry , i just didn't find this field in https://accelbyte.atlassian.net/wiki/spaces/PJ/pages/206307441/Log+Standardization. If you think we still need this, I would add it

File: pkg/logger/event/event.go (hunk 1 above)
Comment (egglee) [reply pkg/logger/event/event.go]: from our wiki, this type should be int.

Comment (thoriqsatriya) [reply pkg/logger/event/event.go]: Actually this const is not used for "event_type" field, but it's used for "log_type". For now I think we still don't want to remove it to keep it compatible with existing log parser.

File: pkg/logger/event/event.go (hunk 2 above)
Comment (thoriqsatriya) [reply pkg/logger/event/event.go]: Yes, I think we still need this. Please add them back. Thank you

File: pkg/logger/event/event.go (hunk 1 above)
Comment (egglee) [reply pkg/logger/event/event.go]: added back now.

File: pkg/logger/event/event.go (hunk 2 above)
Comment (egglee) [reply pkg/logger/event/event.go]: added back.

File: pkg/logger/event/event.go (hunk 3 above)
Comment (egglee) [reply pkg/logger/event/event.go]: I have updated it

**Pull request merged**